README.md

```diff
@@ -97,7 +97,8 @@ A REST call can be executed against one of the cluster instances, and the config
 #### General Optional Parameters
 | Name              | Description                | Default Value  |
 |--------           |----------------------------|-----------------------|
-| `datadog.url` | Datadog url endpoint where your logs will be sent| `http-intake.logs.datadoghq.com:443` ||
+| `datadog.url` | Datadog URL endpoint where your logs will be sent. `datadog.url` takes precedence over `datadog.site`. Example: `http-intake.logs.datadoghq.com:443` ||
+| `datadog.site` | The site of the Datadog intake to send Agent data to (For example 'datadoghq.eu' to send data to the EU site) ||
```

Comment:
Let's explain here that the default for `site` is `datadoghq.com` (even though the actual logic is a little more complex that this, in practice for end users that's the default that matters, right?), and that `datadog.url` is only useful to send to custom URLs, i.e. something like:

```suggestion
| `datadog.site` | The site of the Datadog intake to send logs to (for example 'datadoghq.eu' to send data to the EU site) | `datadoghq.com` |
| `datadog.url` | Custom Datadog URL endpoint where your logs will be sent. `datadog.url` takes precedence over `datadog.site`. Example: `http-intake.logs.datadoghq.com:443` ||
```